### ISSUE TITLE
Rationalize scripts

### ISSUE THREAD

**Issue by karllark** [BEAST-Fitting/beast#305]
There is a growing number of scripts that are used to make plots, run simulation, run the beast, etc.   It would be good to rationalize these - a good topic for a beast hackday discussion.

In addition, they should be moved to the appropriate scripts directory so they are installed in the system when the beast package is installed.  This would mean they can be run from any directory - removing the need for the symlink.  Docs would need updating.  Names of scripts may need updating so that they make sense system wide.

**Comment by lea-hagen** [comment]
I'm starting to think that the beast running part should be worked out for v1.3.  The code is already barely wieldy, and since we're converging on a grid that requires using the subgridding tools, it's only going to get worse.  I want to make sure I'm not the only one who can understand the cobbled-together ensemble of code pieces :).

**Comment by karllark** [comment]
I agree with this sentiment, but I would think we shouldn't make v1.3 dependent on solving this issue.  Mainly to make sure we can get started on production soon.  And in reality, most (or all?) of the 2019 production runs will be done by someone that is part of the ISM@ST group and so you can train them if needed.  

I can assure you that you are not the only one that can run the BEAST.  I during the latest HST proposal season.  :-)

**Comment by karllark** [comment]
This should be done after #407 is done.
